buildmaster/LHCB_DY_13TEV_DIELECTRON/metadata.yaml

```diff
@@ -0,0 +1,53 @@
+# Previously called `LHCB_Z_13TEV_DIELECTRON`
+# author: "Tanjona R. Rabemananjara"
+setname: "LHCB_DY_13TEV_DIELECTRON"
+
+version: 1
+version_comment: "First implementation"
+
+# References
+arXiv:
+  url: "https://arxiv.org/abs/1607.06495"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1477581"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1477581"
+  version: 1
```

Comment:
Since this dataset and `DIMUON` are coming from the same hepdata, I think it would be better to have them both as

`LHCB_DY_13TEV_Y` for instance and have the observables be `DIMUON` `DIELECTRON`

Reply:
Yes, they indeed come the same hepdata. However, in the di-electron production the last bin is removed, so I think these really need to be separate as per https://github.com/NNPDF/nnpdf/pull/1813#issuecomment-1755702965.

Reply:
No, these are different observables so they can have different number of data points (and of course the fktables should be different)

Reply:
So, we agree to leave these as they are currently?

Reply:
No, I mean that you can have more than one observable in a single metadata.yaml
One of them can be `DIMUON` (and have n datapoints) and the other `DIELECTRON` and have a different number `m` of datapoints.

Reply:
Done!